src/index/silentpaymentindex.cpp

```diff
@@ -93,17 +95,11 @@ bool SilentPaymentIndex::GetSilentPaymentKeys(std::vector<CTransactionRef> txs,
 
 bool SilentPaymentIndex::CustomAppend(const interfaces::BlockInfo& block)
 {
-    // Exclude genesis block transaction because outputs are not spendable.
-    if (block.height == 0) return true;
```

Comment:
Removing this is causing an error on all non-mainnet chains now because m_start_height is 0 by default, which is the genesis block and it needs to be skipped because it's unspendable and has no undo data.

Reply:
Brought it back.